<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DWelComePage</class>
 <widget class="QWidget" name="DWelComePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DWelComePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	outline:0px;
	color:#DCDCDC;
}

*:disabled{
	background:#444444;
	border-color:#484848;
}

QMenu{
	color: rgb(20, 20, 20);
	background-color: rgb(218, 218, 218);
}

QWidget{
	border:1px solid #DCDCDC;
	background:#484848;
	border-radius:0px;
	color:#DCDCDC;
	border-style:none;
}

QSplitter::handle{
background:#7c7c7c;
}

QAbstractButton{
	border-radius:0px;
	color:#DCDCDC;
	background:none;
	border-style:none;
}

QAbstractButton:hover{
	color:#FFFFFF;
	background-color:#00BB9E;
}

QAbstractButton:checked,QAbstractButton:pressed{
	color:#DCDCDC;
	border-style:solid;
	border-width:0px 0px 0px 2px;
	padding:4px 4px 4px 2px;
	border-color:#00BB9E;
	background-color:#444444;
}

QLineEdit{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background-color: rgb(90, 90, 90);
	color:#DCDCDC;
	selection-background-color:#484848;
	selection-background-color: rgba(81, 163, 244);
}

QLineEdit:focus{
	border:1px solid #242424;
}

QLineEdit{
	lineedit-password-character:9679;
}

.QGroupBox{
	border:1px solid #242424;
	border-radius:5px;
	margin-top:3ex;
}

.QGroupBox::title{
	subcontrol-origin:margin;
	position:relative;
	left:10px;
}

.QPushButton,.QToolButton{
	border-style:none;
	border:1px solid #242424;
	color:#DCDCDC;
	padding:5px;
	min-height:15px;
	border-radius:5px;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

.QPushButton:hover,.QToolButton:hover{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

.QPushButton:pressed,.QToolButton:pressed{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QRadioButton::indicator{
	width:15px;
	height:25px;
}

QRadioButton::indicator::unchecked{
	image:url(:/images/qss/psblack/radiobutton_unchecked.png);
}

QRadioButton::indicator::unchecked:disabled{	
	image:url(:/images/qss/psblack/radiobutton_unchecked_disable.png);
}

QRadioButton::indicator::checked{
	image:url(:/images/qss/psblack/radiobutton_checked.png);
}

QRadioButton::indicator::checked:disabled{
	image:url(:/images/qss/psblack/radiobutton_checked_disable.png);
}

QGroupBox::indicator,QTreeWidget::indicator,QListWidget::indicator{
	padding:0px -3px 0px 3px;
}

QCheckBox::indicator,QGroupBox::indicator,QTreeWidget::indicator,QListWidget::indicator{
	width:13px;
	height:13px;
}

QCheckBox::indicator:unchecked,QGroupBox::indicator:unchecked,QTreeWidget::indicator:unchecked,QListWidget::indicator:unchecked{
	image:url(:/images/qss/psblack/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:disabled,QGroupBox::indicator:unchecked:disabled,QTreeWidget::indicator:unchecked:disabled,QListWidget::indicator:disabled{
	image:url(:/images/qss/psblack/checkbox_unchecked_disable.png);
}

QCheckBox::indicator:checked,QGroupBox::indicator:checked,QTreeWidget::indicator:checked,QListWidget::indicator:checked{
	image:url(:/images/qss/psblack/checkbox_checked.png);
}

QCheckBox::indicator:checked:disabled,QGroupBox::indicator:checked:disabled,QTreeWidget::indicator:checked:disabled,QListWidget::indicator:checked:disabled{
	image:url(:/images/qss/psblack/checkbox_checked_disable.png);
}

QCheckBox::indicator:indeterminate,QGroupBox::indicator:indeterminate,QTreeWidget::indicator:indeterminate,QListWidget::indicator:indeterminate{
	image:url(:/images/qss/psblack/checkbox_parcial.png);
}

QCheckBox::indicator:indeterminate:disabled,QGroupBox::indicator:indeterminate:disabled,QTreeWidget::indicator:indeterminate:disabled,QListWidget::indicator:indeterminate:disabled{
	image:url(:/images/qss/psblack/checkbox_parcial_disable.png);
}

QComboBox{
	border:1px solid #8f8f8f;
	background-color: rgb(118, 118, 118);
	border-radius: 5px;
}

QComboBox::down-arrow,QDateEdit[calendarPopup=&quot;true&quot;]::down-arrow,QTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow,QDateTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow{
	image:url(:/images/qss/psblack/add_bottom.png);
	width:10px;
	height:10px;
	right:2px;
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:15px;
	border-left-width:0px;
	border-left-style:solid;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	border-left-color:#242424;
}

QComboBox::drop-down:on{
	top:1px;
}

QScrollBar:horizontal{
	background:#484848;
	padding:0px;
	border-radius:6px;
	max-height:12px;
}

QScrollBar::handle:horizontal{
	background:#525252;
	min-width:50px;
	border-radius:6px;
}

QScrollBar::handle:horizontal:hover{
	background:#242424;
}

QScrollBar::handle:horizontal:pressed{
	background:#242424;
}

QScrollBar::add-page:horizontal{
	background:none;
}

QScrollBar::sub-page:horizontal{
	background:none;
}

QScrollBar::add-line:horizontal{
	background:none;
}

QScrollBar::sub-line:horizontal{
	background:none;
}

QScrollBar:vertical{
	background:#484848;
	padding:0px;
	border-radius:6px;
	max-width:12px;
}

QScrollBar::handle:vertical{
	background:#525252;
	min-height:50px;
	border-radius:6px;
}

QScrollBar::handle:vertical:hover{
	background:#242424;
}

QScrollBar::handle:vertical:pressed{
	background:#242424;
}

QScrollBar::add-page:vertical{
	background:none;
}

QScrollBar::sub-page:vertical{
	background:none;
}

QScrollBar::add-line:vertical{
	background:none;
}

QScrollBar::sub-line:vertical{
	background:none;
}

QScrollArea{
	border:0px;
}

QGroupBox::indicator,QTreeWidget::indicator,QListWidget::indicator{
	padding:0px -3px 0px 3px;
}

QListWidget::item:hover{
	background-color: rgb(162, 162, 162);
}

QListWidget::item:selected{
	background-color:rgb(0, 139, 208)
}

QListWidget::item{
	height: 40px;
	width: 60px;
	margin: 1px 2px;
	border:2px solid rgb(0, 117, 176);
}

QTabBar::tab{
border:1px solid #242424;
color:#DCDCDC;
margin:0px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

QTabBar::tab:selected,QTabBar::tab:hover{
border-style:solid;
border-color:#00BB9E;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QTabBar::tab:top,QTabBar::tab:bottom{
padding:3px 8px 3px 8px;
}

QTabBar::tab:left,QTabBar::tab:right{
padding:8px 3px 8px 3px;
}

QTabBar::tab:top:selected,QTabBar::tab:top:hover{
border-width:2px 0px 0px 0px;
}

QTabBar::tab:right:selected,QTabBar::tab:right:hover{
border-width:0px 0px 0px 2px;
}

QTabBar::tab:bottom:selected,QTabBar::tab:bottom:hover{
border-width:0px 0px 2px 0px;
}

QTabBar::tab:left:selected,QTabBar::tab:left:hover{
border-width:0px 2px 0px 0px;
}

QTabBar::tab:first:top:selected,QTabBar::tab:first:top:hover,QTabBar::tab:first:bottom:selected,QTabBar::tab:first:bottom:hover{
border-left-width:1px;
border-left-color:#242424;
}

QTabBar::tab:first:left:selected,QTabBar::tab:first:left:hover,QTabBar::tab:first:right:selected,QTabBar::tab:first:right:hover{
border-top-width:1px;
border-top-color:#242424;
}

QTabBar::tab:last:top:selected,QTabBar::tab:last:top:hover,QTabBar::tab:last:bottom:selected,QTabBar::tab:last:bottom:hover{
border-right-width:1px;
border-right-color:#242424;
}

QTabBar::tab:last:left:selected,QTabBar::tab:last:left:hover,QTabBar::tab:last:right:selected,QTabBar::tab:last:right:hover{
border-bottom-width:1px;
border-bottom-color:#242424;
}

QTreeWidget::indicator{
padding:0px -3px 0px 3px;
}

QTreeWidget::indicator{
	width:13px;
	height:13px;
}

QTreeWidget::branch:closed:has-children{
	margin:4px;
	border-image:url(:/images/qss/psblack/branch_open.png);
}

QTreeWidget::branch:open:has-children{
	margin:4px;
	border-image:url(:/images/qss/psblack/branch_close.png);
}

QTreeWidget::indicator:unchecked{
	image:url(:/images/qss/psblack/checkbox_unchecked.png);
}

QTreeWidget::indicator:unchecked:disabled{
	image:url(:/images/qss/psblack/checkbox_unchecked_disable.png);
}

QTreeWidget::indicator:checked{
	image:url(:/images/qss/psblack/checkbox_checked.png);
}

QTreeWidget::indicator:checked:disabled{
	image:url(:/images/qss/psblack/checkbox_checked_disable.png);
}

QTreeWidget::indicator:indeterminate{
	image:url(:/images/qss/psblack/checkbox_parcial.png);
}

QTreeWidget::indicator:indeterminate:disabled{
	image:url(:/images/qss/psblack/checkbox_parcial_disable.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="mSplitterMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QWidget" name="mWidgetUserInfo" native="true">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="mVerticalSpacerUserInfoUp">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="mWidgetUserInfo_2" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>450</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 110, 110);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="mLabelUserProfile">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="text">
               <string>Profile</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="mLabelUserName">
              <property name="text">
               <string>UserName:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mLineEditUsrName">
              <property name="placeholderText">
               <string>User Name...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mLabelGmail">
              <property name="text">
               <string>GMail:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mLineEditGMail">
              <property name="placeholderText">
               <string>GMail...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="mRoupBoxAccountType">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="title">
             <string>Account type</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="mRiadioBtnTemp">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Temp</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mRadioBtnRegister">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="mLabelLatestLoginTime">
              <property name="text">
               <string>Latest Login:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="mLabelWelcomContent">
              <property name="text">
               <string>Welcome to Japycer-GIS</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>102</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="mVerticalSpacerUserInfoDown">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="mSplitterRightCenter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QSplitter" name="mSplitterRightTop">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>4</number>
       </property>
       <widget class="QWidget" name="mWidgetLatestProjects" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="mLabelLatestProjects">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Latest Open Projects:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="mListWidgetLatestProjs"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mWidgetSummitQuestion" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="mLabelLatestUpdate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Latest Update Contents:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="mListWidgetLatestUpdate"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnContactMe">
             <property name="text">
              <string>Contact Me</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnSupport">
             <property name="text">
              <string>Support</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="mSpliiterRightBottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>4</number>
       </property>
       <widget class="QWidget" name="mWidgetCloudAssets" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="mLabelUsrAssets">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ASSETS:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="mToolBtnRefreshAssets">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../images/images.qrc">
               <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="mTreeWidgetAssets">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mWidgetUploadTask" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="mLabelUploadTask">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>UPLOAD TASK</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="mListWidgetUpLoadTask"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
